web/src/settings_panel_menu.js

```diff
@@ -123,7 +123,7 @@ export class SettingsPanelMenu {
         return true;
     }
 
-    activate_section_or_default(section) {
+    activate_section_or_default(section, right_section = "active-users-admin") {
```

Comment:
I don't think we should keep this default here since the right section value is only needed for one section. We can instead set it below only if `section === user-list-admin`.

Reply:
Ohh I guess we don't need to pass this , since we are passing it as a default for `get_valid_users_section_panel`.